plugins/org.komodo.core/src/org/komodo/repository/ObjectImpl.java

```diff
@@ -68,6 +71,87 @@ ObjectImpl accessOuter() {
     }
 
     /**
+     * Implementation specific and not to be included on the interface!
+     *
+     * @param uow transaction for this operation. Can be null.
+     *
+     * @return the underlying object of this {@link ObjectImpl}
+     *
+     * @throws KException if an error occurs
+     */
+    public Node node(UnitOfWork uow) throws KException {
```

Comment:
If we absolutely need this type of method for testing, I would rename it to something like `testGetJcrNode`. So use a method name that is prefixed with "test" or "testOnly" maybe. One alternative assertion you could do is this:

`assertThat(Repository.get(UnitOfWork uow, String path), is(true));`